ci-operator/step-registry/rosa/tests/runtime/day2/rosa-tests-runtime-day2-commands.sh

```diff
@@ -11,6 +11,14 @@ if [[ ! -z $ROSACLI_BUILD ]]; then
   override_rosacli_build
 fi
 
+if [[ ! -z $COMPUTE_MACHINE_TYPE ]]; then
+  export COMPUTE_MACHINE_TYPE=${COMPUTE_MACHINE_TYPE:-}
+fi
+
+if [[ ! -z $OCM_LOGIN_ENV ]]; then
+  export OCM_LOGIN_ENV=${OCM_LOGIN_ENV:-"staging"}
+fi
+
```

Comment:
Same as above. Also, we do not need export `OCM_LOGIN_ENV`

Reply:
Also in rosa-hcp-advanced-regional profile OCM_LGIN_ENV is used as https://....aws.....staging.com, failing test cases in day2, so I guess it needs it @yasun1